src/xdocs/config_whitespace.xml

```diff
@@ -2055,7 +2055,7 @@ public long toMicros(long d) { return d / (C1 / C0); }
           </tr>
           <tr>
             <td>validateComments</td>
-            <td>If set to true, whitespaces surrounding comments will be validated.</td>
+            <td>Control whenever to validate whitespaces surrounding comments.</td>
```

Comment:
Should `whenever` be `whether`?

Reply:
Thanks a lot. Such stupid mistakes are the most difficult to detect.